src/components/SchemaTable/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react' // eslint-disable-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types' // eslint-disable-line import/no-extraneous-dependencies
+
+function SchemaTable () {
+  return (
+    <div>
+      SchemaTable
+    </div>
+  )
+};
+
+SchemaTable.propTypes = {
+
+}
+
+SchemaTable.defaultProps = {
+
+}
+
```

Comment:
@helfi92 
* **Note about Components**
I've defined the `< SchemaTable />` component as a functional component
so that react-hooks can be utilized for this project.
(reason why `propTypes` and `defaultProps` are defined outside the component)

I've already checked that this works fine with styleguidist,
but wanted to make sure if it's okay to use functional components 
along with hooks for this project. 

I personally think they make the code much cleaner and more readable,
would it be okay to use them throughout this project ?

Reply:
Agreed 💯